workflow/common/convert.go

```diff
@@ -0,0 +1,29 @@
+package common
```

Comment:
this is part of the controller codebase - should you move it to `util/cron/convert.go` maybe?

Reply:
sorry to ask for another change - what about

 ```
func (c *CronWorkflow) ToWorkflow() Workflow {...}
```

Also - unit tests?

Reply:
Will move and rename

Reply:
It would be better for this to stay a utility function so it can be reused easily

Reply:
ok, don't do the ToWorkflow bit, keep as a utility func